tests/test_halo.py

```diff
@@ -441,6 +442,25 @@ def test_animation_setter(self):
         spinner.animation = "marquee"
         self.assertEquals("marquee", spinner.animation)
 
+    def test_spinner_color(self):
+        """Test ANSI escape characters are present
+        """
+        colors = {
+            'cyan': Fore.CYAN,
+            'magenta': Fore.MAGENTA,
+            'yellow': Fore.YELLOW,
+            'blue': Fore.BLUE
+        }
+
+        for k, v in colors.items():
```

Comment:
@eoinnoble Lets make this:

```suggestion
        for color_name, color_ascii in colors.items():
```